doc/nrf/known_issues.rst

```diff
@@ -81,13 +82,25 @@ NCSDK-5574: Warnings during FOTA
 .. rst-class:: v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0 v0-4-0 v0-3-0
 
 NCSDK-6689: High current consumption in Asset Tracker
-  The :ref:`asset_tracker` sample might show up to 2.5 mA current consumption in idle mode with ``CONFIG_POWER_OPTIMIZATION_ENABLE=y``.
+  The :ref:`asset_tracker` application might show up to 2.5 mA current consumption in idle mode with ``CONFIG_POWER_OPTIMIZATION_ENABLE=y``.
 
 .. rst-class:: v1-0-0 v0-4-0 v0-3-0
 
 Sending data before connecting to nRF Cloud
-  The :ref:`asset_tracker` sample does not wait for connection to nRF Cloud before trying to send data.
-  This causes the sample to crash if the user toggles one of the switches before the board is connected to the cloud.
+  The :ref:`asset_tracker` application does not wait for connection to nRF Cloud before trying to send data.
+  This causes the application to crash if the user toggles one of the switches before the board is connected to the cloud.
+
+.. rst-class:: v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0 v0-4-0 v0-3-0
+
```

Comment:
Two blank lines - I don't think this will cause issues, but better to fix.

Reply:
Yes, fixed now.